src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Why/WhyCommandArgs.cs

```diff
@@ -22,16 +24,19 @@ internal class WhyCommandArgs
         /// <param name="package">The package for which we show the dependency graphs.</param>
         /// <param name="frameworks">The target framework(s) for which we show the dependency graphs.</param>
         /// <param name="logger"></param>
+        /// <param name="cancellationToken"></param>
         public WhyCommandArgs(
             string path,
             string package,
             List<string> frameworks,
-            ILoggerWithColor logger)
+            ILoggerWithColor logger,
+            CancellationToken cancellationToken)
         {
             Path = path ?? throw new ArgumentNullException(nameof(path));
             Package = package ?? throw new ArgumentNullException(nameof(package));
             Frameworks = frameworks ?? throw new ArgumentNullException(nameof(frameworks));
             Logger = logger ?? throw new ArgumentNullException(nameof(logger));
+            CancellationToken = cancellationToken; // can't use null-coalescing because CancellationToken is a struct
```

Comment:
```suggestion
            CancellationToken = cancellationToken; // can't use null-coalescing because CancellationToken is a struct
```

Probably don't need the comment. Feels like one of those things that most people know and the rest eventually figure out.